docs/how-to/configure.md

```diff
@@ -60,17 +60,48 @@ Values for the *Custom* preset can be entered as values, or changed with the arr
 
 ### Configure metrics resolution per-service
 
-While changing the metrics resolution in the settings tab applies to all services, you can also configure metrics resolutions on a per-service basis by setting the resolution settings for each exporter individually via the API.
+You can configure metrics resolutions both globally and on a per-service basis. While the global setting in the **Settings** tab applies to all services,  PMM 2.42 and later also enables you to customize resolution for individual services. You can do this by adjusting the `metrics_resolutions` setting for each exporter using the "[Change Agent Attributes](https://percona-pmm.readme.io/reference/changeagent)" API (see below for an example).
 
-Customizing the resolution settings for individual services enables you to fine-tune your PMM setup to balance data granularity and resource consumption.
-
-This enables you to:
+Customizing resolution settings for individual services allows you to fine-tune your PMM setup, balancing data granularity with resource consumption. This feature enables you to:
 
 - Allocate resources efficiently by focusing on high-resolution data for key services
-- Reduce storage requirements by adjusting resolution for less critical components
+- Reduce storage requirements by lowering resolution for less important components
 - Align your monitoring setup with the specific needs of your environment
-
-For information on enabling this feature via API, see the [**Change Postgres Exporter endpoint**](https://percona-pmm.readme.io/reference/changepostgresexporter) in the API documentation.
+  
+To change resolution settings:
+
+1. Identify the appropriate API endpoint:
+
+     - For PostgreSQL: `ChangePostgresExporter`
+     - For MySQL: `ChangeMySQLdExporter`
+     - For MongoDB: `ChangeMongoDBExporter`
+
+2. Locate the `agent_id` of the exporter you want to modify. You can find this in the [Inventory dashboard](../details/dashboards/dashboard-inventory.md) under the **Monitoring** column for the target service.
+   
+4. Set the desired resolution using `hr` (high), `mr` (medium), or `lr` (low) in the `metrics_resolutions` field.
+ 
+**Example**:  Setting 15s medium resolution for a PostgreSQL server:
+
+```
+> curl -X 'POST' \
+  'http://admin:admin@127.0.0.1/v1/inventory/Agents/ChangePostgresExporter' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "agent_id": "/agent_id/64b195b1-4d65-4838-9eab-50b79c0a07a3", // Your exporter agent ID
+  "common": {
+    "enable": true,
+    "disable": true,
+    "custom_labels": {
+      "metrics_resolutions": {"mr": "15s"} // it can be 'mr', 'hr, 'lr' in any combination
+    }
+  }
+}'
+```
+
+To reset a custom resolution, make an API call with "mr": "0s" (or "hr": "0s", "lr": "0s" as appropriate). This will revert the exporter to using the global PMM settings.
```

Comment:
To add a sentence also to inform users how they can "verify" that the change took effect, after sending the API request.  This is done by checking the `scrape_interval` in the promscrape.config file, as far as I understand

Reply:
now done. resolving.